pkg/apis/sources/config/pingconfig/config/doc.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// +k8s:deepcopy-gen=package
+
+// Package config holds the typed objects that define the schemas for
+// ConfigMap objects that pertain to our API objects.
+package config
```

Comment:
can we move all files in this package into `apis/source/config`?

Reply:
yeah +1 on relocating all of those

Reply:
What if we have future sources with a config? The thought behind doing this was to allow pkg/apis/sources/config/somenewsource/... ?

Reply:
flattening does not prevent adding defaults for other sources, does it?

Reply:
Probably not.  And I'll go ahead on do this change. I'm on the record :) that keeping them separate while _maybe_ not necessary would be cleaner IMO